app/controllers/users.js

```diff
@@ -285,7 +285,7 @@ class UsersController extends BaseController {
       });
     }
 
-    await LastActivityiesController.statusUnsubscribe(ws, {
+    await LastActivityiesController.status_unsubscribe(ws, {
```

Comment:
hmm, why did you choose snake_case for methods naming?

Reply:
From the very beginning, basic controllers used this way of naming. And I thought that for basic functioning it would be easier to read words through a space than when they are written solid